phpstan.neon

```diff
@@ -1,6 +1,10 @@
 parameters:
   excludes_analyse:
     - */tests/*/data/*
+    - */Full2Test.php
+    - */Full4Test.php
```

Comment:
Why exclude these?

Reply:
In v4 the interface changed (removed setters, parameter in contructor). Phpstan will complain this.